bitrise.yml

```diff
@@ -272,19 +271,16 @@ workflows:
             exec unzip "$derived_data"
             echo "show dir"
             ls
-    - script@1.1:
+    - xcode-test-without-building@0:
         run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
         title: UI Smoketest3
         inputs:
-        - content: |
-            #!/usr/bin/env bash
-            set -ex
-            ls ./Users/vagrant/git/Tests/
-            cp -r ./Users/vagrant/git/Tests/* .
-            mv ./Users/vagrant/git/DerivedData .
-            ls -la
-            echo "-- test-without-building --"
-            xcodebuild -resultBundlePath "xcodebuild.xcresult" -derivedDataPath "/Users/vagrant/git/DerivedData" -destination "platform=iOS Simulator,name=iPhone 14,OS=16.4" test-without-building -xctestrun "/Users/vagrant/git/DerivedData/Build/Products/Fennec_Smoketest3_iphonesimulator16.4-x86_64.xctestrun"
+        - destination: platform=iOS Simulator,name=iPhone 14 Plus,OS=16.4
+        - xcodebuild_options: '"-derivedDataPath" "/Users/vagrant/git/DerivedData"
+            "-parallel-testing-enabled" "YES" "-parallel-testing-worker-count" "2"
+            "COMPILER_INDEX_STORE_ENABLE=NO" "CODE_SIGN_IDENTITY=" 
+            "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO"'
+        - xctestrun: "Users/vagrant/git/DerivedData/Build/Products/Fennec_Smoketest3_iphonesimulator16.4-x86_64.xctestrun"
     - custom-test-results-export@0:
```

Comment:
Same as above, you should remove this step in favor of the deploy step I mention, with the correct inputs. (you can re-use the existing deploy here in L293